README.md

```diff
@@ -1,3 +1,19 @@
+# SPELLBOOK FREEZE
+
+From June 22 to June 29, we will be freezing some Sepllbook contributions for the migration to DuneSQL. We will not accept contributions to the lineage of the following models:
+
+* dex.trades
+* nft.trades
+* labels
```

Comment:
maybe we add 
* `tokens.nft`
* `tokens.erc20`

as these receive regular contributions and are in the lineage as well.

Reply:
agreed, even if they appear in `dbt ls` command, it'll be a long list and can easily be glossed over

Reply:
Good call. Adding it now